pr_consistency/4.check_consistency.py

```diff
@@ -17,8 +17,8 @@ def parse_isoformat(string):
 
 
 # Only consider PRs merged after this date/time. At the moment, this is the
-# date and time at which the v1.0.x branch was created.
-START = parse_isoformat('2015-01-27T16:22:59')
+# date when the v4.0.x branch was created.
+START = parse_isoformat('2019-11-10T00:00:00')
```

Comment:
No point going too far in the past, as we're not going to fix any pre-v4.0.x warnings.

Reply:
This is OK here, but it's a reminder that these scripts are used beyond the core package, at least I use these whenever I release anything else, so maybe just update the comment that we won't fix ancient cases rather than refer to an astropy branch.

Reply:
Ah good point! Maybe I could make the date choice conditional on the repo?

Reply:
Sure,  but I feel it's totally fine to hardwire something closer to now, e.g. 2-3 years ago is safely in the past that we unlikely want to fix issues for any of those old merges.